docs/docs/how-to-guides/covid_cleaning_exploration.ipynb

```diff
@@ -102,7 +102,11 @@
   {
    "cell_type": "code",
    "execution_count": 164,
-   "metadata": {},
+   "metadata": {
```

Comment:
Please double check that you don't introduce code changes in the documentation notebooks